install.sh

```diff
@@ -40,7 +40,10 @@ apt-get install -y  \
   rsync=3.1.0* \
   psmisc=22.20* \
   vim=2:7.4.052*
+  
+# They not resolved till now - https://github.com/Microsoft/WSL/issues/2782
 
+# This types of warnings is negligible.If we upgrade python it wont show this type of warnings
```

Comment:
`python throws a few warnings that can be ignored. New versions of python do not throw these warnings`